dbms/src/TestUtils/mockExecutor.cpp

```diff
@@ -393,6 +393,24 @@ DAGRequestBuilder & DAGRequestBuilder::expand(MockVVecColumnNameVec grouping_set
     return *this;
 }
 
+DAGRequestBuilder & DAGRequestBuilder::appendRuntimeFilterIds(const std::vector<int> & rf_ids) {
```

Comment:
can we add a new param in scan and join to append rf？
IMHO, It will be cleaner.